notify/ses.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env sh
+
+#Support Amazon SES api
+
+#AWS_ACCESS_KEY=""
+#AWS_SECRET_KEY=""
+#AWS_REGION=""
+#AWS_SES_TO="xxxx@xxx.com"
+#AWS_SES_FROM="xxxx@cccc.com"
+
+ses_send() {
+  _subject="$1"
+  _content="$2"
+  _statusCode="$3" #0: success, 1: error 2($RENEW_SKIP): skipped
+  _debug "_statusCode" "$_statusCode"
+
+  AWS_ACCESS_KEY="${AWS_ACCESS_KEY:-$(_readaccountconf_mutable AWS_ACCESS_KEY)}"
+  if [ -z "$AWS_ACCESS_KEY" ]; then
+    AWS_ACCESS_KEY=""
+    _err "You didn't specify a amazon access key AWS_ACCESS_KEY yet."
+    _err "See https://docs.aws.amazon.com/en_us/general/latest/gr/aws-sec-cred-types.html"
+    return 1
+  fi
+  _saveaccountconf_mutable AWS_ACCESS_KEY "$AWS_ACCESS_KEY"
+
+  AWS_SECRET_KEY="${AWS_SECRET_KEY:-$(_readaccountconf_mutable AWS_SECRET_KEY)}"
+  if [ -z "$AWS_SECRET_KEY" ]; then
+    AWS_SECRET_KEY=""
+    _err "You didn't specify a amazon secret key AWS_SECRET_KEY yet."
+    _err "See https://docs.aws.amazon.com/en_us/general/latest/gr/aws-sec-cred-types.html"
+    return 1
+  fi
+  _saveaccountconf_mutable AWS_SECRET_KEY "$AWS_SECRET_KEY"
+
+  AWS_REGION="${AWS_REGION:-$(_readaccountconf_mutable AWS_REGION)}"
+  if [ -z "$AWS_REGION" ]; then
+    AWS_REGION=""
+    _err "You didn't specify the AWS_REGION."
+    return 1
+  fi
+  AWS_REGION="$(echo "$AWS_REGION" | _lower_case)"
+  _saveaccountconf_mutable AWS_REGION "$AWS_REGION"
+
+  AWS_SES_TO="${AWS_SES_TO:-$(_readaccountconf_mutable AWS_SES_TO)}"
+  if [ -z "$AWS_SES_TO" ]; then
+    AWS_SES_TO=""
+    _err "You didn't specify an email to AWS_SES_TO receive messages."
+    return 1
+  fi
+  _saveaccountconf_mutable AWS_SES_TO "$AWS_SES_TO"
+
+  AWS_SES_FROM="${AWS_SES_FROM:-$(_readaccountconf_mutable AWS_SES_FROM)}"
+  if [ -z "$AWS_SES_FROM" ]; then
+    AWS_SES_FROM=""
+    _err "You didn't specify an email to AWS_SES_FROM receive messages."
+    return 1
+  fi
+  _saveaccountconf_mutable AWS_SES_FROM "$AWS_SES_FROM"
+
+  _date="$(date -R)"
+  _signature="$(echo -n "$_date" | ${ACME_OPENSSL_BIN:-openssl} dgst -sha256 -hmac "$AWS_SECRET_KEY" -binary | base64 -w 0)"
```

Comment:
don't use  `openssl`,  we have `_hmac()` function for use.

Here are good examples:

https://github.com/Neilpang/acme.sh/blob/master/dnsapi/dns_aws.sh#L298

Reply:
Changed it to use _hmac and _base64 functions. Works fine on my side, but there is one question regarding to variable naming. 
Here I used AWS_ACCESS_KEY and AWS_SECRET_KEY as variable names, the dns_aws.sh script uses AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY as the same variable names. Should I use the same variable names in both scripts or will it be better to make them different (SES account may differ from route53 account)?

Reply:
Please use the same name: AWS_ACCESS_KEY_ID 

The name was borrowed from AWS official SDK,   that means, the user doesn't need to set it explicitly.  We can get the value automatically.

Reply:
yes, please use the same name.

Reply:
Done)